src/util/num64.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+// Copyright 2014 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include <stdint.h>
```

Comment:
I think the portable way is to use `#include <cstdint>` instead (according to https://en.cppreference.com/w/cpp/types/integer )